Gothic-Unity-Project/Gothic-Unity
# Feature/npc spawn on top of each other

> Please test now, but merge only after https://github.com/Gothic-UnZENity-Project/Gothic-UnZENity/pull/39

# To test:
1. Start game with all NPCs active
2. Stop immediately after they are spawned
3. Check the locations I mentioned in screenshots if the NPCs spawn _next to each other_ instead of _onto each other_

OC: Mages
![image](https://github.com/Gothic-UnZENity-Project/Gothic-UnZENity/assets/120568393/cc7902ab-a603-4234-bf74-0247deff9593)

Bloodwyn and Ball Taran:
![image](https://github.com/Gothic-UnZENity-Project/Gothic-UnZENity/assets/120568393/0575ff85-413e-45c0-822d-5fbde4bcb613)


Hint:
* If you compare to G1 closely, you will see, that the rotation angle of where to look for free spots is different. But tbh: who cares. 🤷 
![image](https://github.com/Gothic-UnZENity-Project/Gothic-UnZENity/assets/120568393/8f9dfeeb-db93-4cff-b951-349ba62d4380)
